examples/calculator/lib/main.dart

```diff
@@ -0,0 +1,258 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'calc_expression.dart';
```

Comment:
Please add a blank line between `package` imports and path-relative imports.

See https://flutter.io/style-guide/ for the complete style guide.